test/support/router.ex

```diff
@@ -17,6 +17,9 @@ defmodule Phoenix.LiveViewTest.Router do
   scope "/", Phoenix.LiveViewTest do
     pipe_through [:browser]
 
+    # controller test
+    get "/clock_controller", ClockController, :index
```

Comment:
If there's a way to test it without actually creating a controller, I would love to know ❤️